TECHMANIA/Assets/Editor/BuildPostProcessor.cs

```diff
@@ -0,0 +1,22 @@
+using System.IO;
+using UnityEngine;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEditor.iOS.Xcode;
+public class BuildPostProcessor
+{
+    [PostProcessBuild]
+    public static void ChangeXcodePlist(BuildTarget buildTarget, string path)
+    {
+        if (buildTarget == BuildTarget.iOS)
+        {
+            string plistPath = path + "/Info.plist";
+            PlistDocument plist = new PlistDocument();
+            plist.ReadFromFile(plistPath);
+            PlistElementDict rootDict = plist.root;
+            rootDict.SetBoolean("UIFileSharingEnabled", true);
+            rootDict.SetBoolean("UISupportsDocumentBrowser", true);
+            File.WriteAllText(plistPath, plist.WriteToString());
```

Comment:
Can we do `plist.WriteToFile` so it's symmetric with `plist.ReadFromFile`?

Reply:
This should be fixed in https://github.com/techmania-team/techmania/pull/9/commits/13c9f37c6d0aa2ccf904c9a039e733ab088bf606